notebooks/tests/test2_tellseq_C_equal_volume_pooling.py

```diff
@@ -0,0 +1,159 @@
+import unittest
+import papermill as pm
+import tempfile
+from pathlib import Path
+import os
+import shutil
+
+NOTEBOOK = "tellseq_C_equal_volume_pooling.ipynb"
+
+# Golden files live under notebooks/test_output/<subdir>/<filename>
+EXPECTED_SUBDIRS = ["Pooling", "Indices", "SampleSheets", "QC"]
+
+# Only compare data files (skip .ipynb, images, etc.)
+DATA_EXTS = {".txt", ".csv", ".tsv"}
+
+
+class TestTellseqC_ShadowWorkspace(unittest.TestCase):
+    def setUp(self):
+        # project root (parent of this test file’s folder)
+        self.notebooks_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+        self.repo_test_output_dir = os.path.join(self.notebooks_dir, "test_output")
+
+        # sanity: required inputs exist in repo
+        must_inputs = [
```

Comment:
This is very careful programming, and I can see that you are looking out for error cases.  However, since this is NOT a user-facing tool (these unit tests will be run by the code developers only), it is not necessary to employ this level of caution (and complication).  We can take the python approach of Easier to Ask Forgiveness than Permission (EAFP, see https://realpython.com/ref/glossary/eafp/ ) and just let the code fail if it tries to access the input files and they aren't there.  The unit test framework already has error-catching baked in, and the built-in python error thrown in the case of a missing file will be informative enough for a programmer to figure out what is going on :)